# DP: Link using --hash-style=gnu (aarch64).

2012-11-17  Matthias Klose  <doko@ubuntu.com>

	* config/aarch64/aarch64-linux.h (LINK_SPEC): Add --hash-style=gnu.
	
Index: b/src/gcc/config/aarch64/aarch64-linux.h
===================================================================
--- a/src/gcc/config/aarch64/aarch64-linux.h
+++ b/src/gcc/config/aarch64/aarch64-linux.h
@@ -32,6 +32,7 @@
 #define GLIBC_DYNAMIC_LINKER "/lib/ld-linux-aarch64.so.1"
 
 #define LINUX_TARGET_LINK_SPEC  "%{h*}		\
+   --hash-style=gnu				\
    %{static:-Bstatic}				\
    %{shared:-shared}				\
    %{symbolic:-Bsymbolic}			\
